src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,21 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { FC } from 'react';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: FC<Props> = ({
+  todo,
+}) => (
+  <article
+    className={
+      `TodoInfo
+      ${todo.completed ? ' TodoInfo--completed' : ''}`
+    }
```

Comment:
use classNames for this
```suggestion
    className={
      classNames('TodoInfo', ...`
    }
```